test/fail_compilation/misc_parser_err_cov1.d

```diff
@@ -17,7 +17,7 @@ fail_compilation/misc_parser_err_cov1.d(38): Error: found `)` instead of stateme
 fail_compilation/misc_parser_err_cov1.d(39): Error: identifier expected following `(type)`.
 fail_compilation/misc_parser_err_cov1.d(39): Error: expression expected, not `;`
 fail_compilation/misc_parser_err_cov1.d(40): Error: semicolon expected following auto declaration, not `auto`
-fail_compilation/misc_parser_err_cov1.d(40): Error: identifier expected following `.`, not `+`
+fail_compilation/misc_parser_err_cov1.d(40): Error: identifier or `new` expected following `.`, not `+`
```

Comment:
Not sure where the semantic change is in the parser (it's not just a refactor).

Reply:
The message error was near an `else` that's been removed (line 8450) and I've noticed that it was incomplete in comparison with what was tested before. There's no semantic change.

Reply:
Thanks for the explanation. Since it changes the output of the compiler, it is a semantic change. In general, such should be in separate PRs, because it makes them easier to review and easier to track down bugs introduced by refactoring.

This one is indeed minor, so in the interests of expeditiousness, I'll approve.